geode-apis-compatible-with-redis/src/test/java/org/apache/geode/redis/internal/publishAndSubscribe/SubscriptionsJUnitTest.java

```diff
@@ -318,8 +319,74 @@ public void findChannelNames_shouldNotReturnDuplicates_givenMultipleSubscription
     subject.add(subscriptionFoo1);
     subject.add(subscriptionFoo2);
 
-    List<byte[]> result = subject.findChannelNames(Coder.stringToBytes("f*"));
+    List<Object> result = subject.findChannelNames(Coder.stringToBytes("f*"));
 
     assertThat(result).containsExactlyInAnyOrder(Coder.stringToBytes("foo"));
   }
+
+  @Test
+  public void findNumberOfSubscribersByChannel_shouldReturnListOfChannelsAndSubscriberCount_givenListOfActiveChannels() {
+    Subscriptions subject = new Subscriptions();
+
+    Channel channel = mock(Channel.class);
+    when(channel.closeFuture()).thenReturn(mock(ChannelFuture.class));
+    Client client = new Client(channel);
+    ExecutionHandlerContext context = mock(ExecutionHandlerContext.class);
+
+    Subscription subscriptionFoo1 =
+        new ChannelSubscription(client, Coder.stringToBytes("foo"), context, subject);
```

Comment:
Can the byte arrays/strings in these tests be extracted to variables (when they're referred to in multiple places)?